### ISSUE TITLE
Invalid "Error, item is duplicate" error message when a changed line is reverted.

### ISSUE THREAD

**Issue by ericpromislow** [rancher-sandbox/rancher-desktop#5439]
### Actual Behavior

Enable the 'Allowed Images Patterns' field and add the pattern "abcde", press return.

Now edit the line, delete the "e", and then retype it.

You'll get the "Error, item is duplicate"  message.

The message itself needs to be revised. First, the comma should be a colon (`:`). And I would change the text to be something like "the current line is a duplicate" or "the current line is already entered".

### Steps to Reproduce

.

### Result

.

### Expected Behavior

.

### Additional Information

.

### Rancher Desktop Version

1.10.0rc1

### Rancher Desktop K8s Version

f

### Which container engine are you using?

containerd (nerdctl)

### What operating system are you using?

macOS

### Operating System / Build Version

f

### What CPU architecture are you using?

x64

### Linux only: what package format did you use to install Rancher Desktop?

None

**Comment by rak-phillip** [comment]
This fails in the 1.11-RC1 test case RD-67: Test case 67: Image Allow List - component behavior and design

**Comment by jandubois** [comment]
> The message itself needs to be revised.

I agree that the message needs to be revised, but either I misunderstand the repro steps, or this should not throw an error at all.

IIUC you are editing a line, and then you undo the edit. Then you get an error that the line is a duplicate (of itself, which doesn't make any sense). Or are there actually any duplicate lines in this repro?

To me it sounds like the line being edited needs to be skipped when checking for duplicates.
